packages/itwin/reports-config-widget/src/widget/components/ReportMappingHorizontalTile.tsx

```diff
@@ -26,54 +26,56 @@ export interface ReportMappingHorizontalTileProps {
 
 export const ReportMappingHorizontalTile = (props: ReportMappingHorizontalTileProps) => {
   const [extractionState, setExtractionState] = useState<ExtractionStates>(ExtractionStates.None);
-  const [jobStarted, setJobStarted] = useState<boolean>(true);
   const interval = useRef<number>();
-  const initialLoad = useRef<boolean>(true);
 
   useEffect(() => {
     const listener = (startedIModelId: string) => {
       if (startedIModelId === props.mapping.imodelId) {
         setExtractionState(ExtractionStates.Starting);
-        setJobStarted(true);
       }
     };
     props.jobStartEvent.addListener(listener);
 
     return () => {
       props.jobStartEvent.removeListener(listener);
     };
-  }, [props.jobStartEvent, props.mapping]);
+  }, [props.jobStartEvent, props.mapping.imodelId]);
 
   const getExtractionState = useCallback(async () => {
```

Comment:
This callback needs renaming since `get` implies that it will return the extraction state but refreshes the react component state.

Reply:
Good call. Changed.